index.bs

```diff
@@ -1534,6 +1535,17 @@ terminates the operation and returns an error.
 
 If the user refuses consent, the authenticator returns an appropriate error status to the client.
 
+When this method is invoked on a credential with ScopedCredentialType "ScopedCred_FIDO_UAF", the authenticator must perform the procedure as specified in [[FIDO-UAF-Authenticator-Commands]] section 6.3.  
```

Comment:
Format the two "When this method is" sections using `<dl class=switch>` to be clear where the boundary between the two options is. Something like:

```
1. If the quabbledyblargh is:
    <dl class="switch">
    <dt>"public-key"</dt>
    <dd>
    1. Check if all the supplied parameters …
    </dd>
    <dt>"public-key-uaf"</dt>
    <dd>
    1. perform the procedure as specified in [FIDO-UAF-Authenticator-Commands] section 6.3. 
    2. …
    </dd>
    </dl>
```

You'll have to fill in "quabbledyblargh" with some value computed from the arguments to this operation, which don't actually seem to have enough information in them to identify which algorithm to run. So … figure that out. 😃

You'll also have to play with the indentation and blank lines to get Markdown happy with that `<dl>`. I think there are some other examples you can try to imitate.